src/protocol/kn_nrf24l01.c

```diff
@@ -562,7 +563,7 @@ static void kn_update_packet_control_data(u8 packet[], s32 packet_count, s32 rf_
     } else {
        packet[9]  = Channels[CHANNEL10] /101 + 100; // 0x64; // A
        packet[10] = Channels[CHANNEL11] /101 + 100; // 0x64; // E
-       packet[11] = 0x64; // R
+       packet[11] = Channels[CHANNEL12] /101 + 100; // 0x64; // R
```

Comment:
Lint Error:
```
566:  At least two spaces is best between code and comments  
```

Reply:
My line is the same as the existing ones

Reply:
That's because there was no Lint check yet when this file was first commited.

Reply:
What to do then? Should change all the file commets?

Reply:
At least the one on the line you modified